### ISSUE TITLE
Small changes (no high priority)

### ISSUE THREAD

**Issue by karpaddel** [hydrocode-de/integra-frontend#154]
Here we gathered some small issues, which we would like to change. They do not have high priority (do not need to be implemented before the presentation):

- **German species names:** Generally, we think that most users will be more familiar with the German species names -> Latin names could only appear when selecting tree species and in the "Steckbriefe" otherwise we would change the names to German. (Here we would like to ask Chris again, before changes are made)

- **Tree info:** When hovering over a tree in the map, a window appears with the species name and some info about its age and height. We feel like the font size of the species name is quite large, this could be equally or only a bit larger-sized than the rest of the text here. Also, better the German name here. And, the info about age and height should also appear in German. Height values should be rounded to a max. of 1 decimal digit.

- **Color of tree stripe:** the color of the tree stripe borders is currently orange. We don't like, that there are in total two different colors for the stripes (green and orange). That is why we suppose to change the orange to the same color as the stripe filling (dark green), but to differentiate it from the transparent stripe, we would not make it transparent here.

- **Color of area border:** here there are two colors (light and dark pink), which we would like to reduce to only one color, preferably a less harsh rose color (rosa statt pink).

- **Possibility to place trees outside of the area:** This is currently possible, but shouldn't be. 

- **Distances:** The front size could still be a little larger. Sometimes, the distance between two trees is displayed double and is overlapping, this does not look very nice, maybe this could be changed.

**Comment by karpaddel** [comment]
- [ ] **North arrow:** the north arrow is currently quite discreet. Would it be possible, that the arrow is a seperate symbol that is not directly attached to the plus and minus (similar to Google maps)? And we would like to change the darker color to Red.

- [ ] **Shrub-Icons:** these look quite large. Can they be downsized, so the 
relation to the tree size is more appropriate?

- [ ] **Insect-tab:** For the first plot, different insect species can be selected. In brackets the info on pollen/larvae is given. Here the word 'Pollen' is missing, it should be: "Erdhummel (XY mm³ Pollen / Larve)"

- [ ] **Tree planning:** In the card that pops-up when clicking on a tree (tree planning), there is still the star grading for "Schatten" and "Blühangebot". This should be removed here and only be present within the "Steckbriefe" in the tree species selection.

**Comment by JoshuaBWWD** [comment]
- [ ] **Übersicht:**: when minimizing the window with "Biomasse, Schatten, Blüten, Bienen", please change the **name** from "Übersicht" to "Leistungsübersicht";
- [ ] **Copying tree rows**: for easier handling, it would usefull to copy a tree row and move it as such, without moving each tree separately.

**Comment by noobla11** [comment]
@mmaelicke Here are some tasks which are already solved I think? Such as "Tree planning" and "German species name". Maybe you could indaicate the completet tasks for clarity :)

**Comment by mmaelicke** [comment]
yep, just did that. Actually not sure if I replaced the species name everywhere...

The point about copying a tree-line is not a small change, though, but a major development. While copying is something that I can somehow hack in somehow, the moving part is currently close to impossible.
That was one of the things that I pointed out, when moving from tree-lines to individual trees as the base data model. If moving entire tree lines is important, we need yet another extra issue for that and I have to think about a possible implementation. 
Anyway, both will not happen until 15.7.

**Comment by mmaelicke** [comment]
> * [ ]  **Shrub-Icons:** these look quite large. Can they be downsized, so the
>   relation to the tree size is more appropriate?
As far as I can tell, the sizes largely match. There are sometimes small inconsistencies, when the zoom is not perfectly aligned. Additionally, there is a minimum size implemented, so that icons are always visible. That can affect shrubs, that are below this minimum size for a couple of years. But unfortunately, there is no way around that. I think the smallest icons should not get any smaller...

**Comment by noobla11** [comment]
> yep, just did that. Actually not sure if I replaced the species name everywhere...
> 
> The point about copying a tree-line is not a small change, though, but a major development. While copying is something that I can somehow hack in somehow, the moving part is currently close to impossible. That was one of the things that I pointed out, when moving from tree-lines to individual trees as the base data model. If moving entire tree lines is important, we need yet another extra issue for that and I have to think about a possible implementation. Anyway, both will not happen until 15.7.

Okay thank you for elaborating again. I will open another issue and we go from there.

**Comment by noobla11** [comment]
> > * [ ]   **Shrub-Icons:** these look quite large. Can they be downsized, so the
> >   relation to the tree size is more appropriate?
> 
> As far as I can tell, the sizes largely match. There are sometimes small inconsistencies, when the zoom is not perfectly aligned. Additionally, there is a minimum size implemented, so that icons are always visible. That can affect shrubs, that are below this minimum size for a couple of years. But unfortunately, there is no way around that. I think the smallest icons should not get any smaller...

I think we can ditch this task.
